homeassistant/components/eafm/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""UK Environment Agency Flood Monitoring Integration."""
+
+from homeassistant.helpers import device_registry as dr
+
+from .const import DOMAIN
+from .station import Station
+
+
+async def async_setup(hass, config):
+    """Set up devices."""
+    hass.data[DOMAIN] = {}
+    return True
+
+
+async def async_setup_entry(hass, entry):
+    """Set up flood monitoring sensors for this config entry."""
+    station_key = entry.data["station"]
+
+    device_registry = await dr.async_get_registry(hass)
+    device_registry.async_get_or_create(
+        config_entry_id=entry.entry_id,
+        identifiers={(DOMAIN, "station-id", station_key)},
+        name=entry.title,
+        manufacturer="https://environment.data.gov.uk/",
+        model="Real Time flood-monitoring API",
+    )
+
+    station = hass.data[DOMAIN][station_key] = Station(hass, entry)
+    hass.add_job(station.async_start)
```

Comment:
Please use `hass.async_create_task` instead of `hass.add_job`.

Reply:
Did this but now deleted!